File: src/main/java/com/clearcont/clearcontapp/model/ComposicaoDePendencia.java
package com.clearcont.clearcontapp.model;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDateTime;

@Entity
@Setter
@Getter
public class ComposicaoDePendencia {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer id;
    private LocalDateTime data;
    private String descricao;
    private Double valor;
    private String areaResposavel;
    private String observacoes;
    private String aging;
}

File: src/main/kotlin/br/com/clearcont/clearcontwebapp/models/ComposicaoDePendencia.kt
package br.com.clearcont.clearcontwebapp.models

import jakarta.persistence.*
import java.time.LocalDateTime

@Entity
class ComposicaoDePendencia {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    val id: Long? = null
    val data: LocalDateTime? = null
    val descricao: String? = null
    val valor: Double? = null
    @Column(length = 30)
    var areaResposavel: String? = null
        private set
    val observacoes: String? = null
    @Column(length = 30)
    var aging: String? = null
        private set
}
